new/LIT7211LaBeesiSemue.xml

```diff
@@ -0,0 +1,116 @@
+                        verses is <term key="EtanaMogar">ʿǝṭāna mogar</term> sung in <term key="EzelMode">ʿǝzl</term> mode.</bibl>
+                        </p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="EtanaMogar">ʿǝṭāna mogar ʿǝzl</term>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2024-12-20">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+        <div type="bibliography">
+            <listRelation>
+                <relation name="lawd:hasAttestation" active="LIT7211LaBeesiSemue" passive="BNFabb145"/>
+            </listRelation>        
+            
+            <listBibl type="editions">
+                <bibl>
+                    <ptr target="bm:Afawarq1984Yageezqeneyat"/>
+                    <citedRange unit="item">76</citedRange>
+                    <citedRange unit="page">77-78</citedRange>
+                </bibl>
+            </listBibl>
+        </div>
+            
+            <div type="edition">
+                <div type="textpart" xml:lang="gez">
+                    <note>The text is quoted after<bibl><ptr target="bm:Afawarq1984Yageezqeneyat"/>
+                        <citedRange unit="item">76</citedRange></bibl>.</note>
+                    <ab>                                              
+                        <l n="1">ለብእሲ፡ ጽሙዕ፡ ኢያረውዮ፡ ቈሪረ፡ ማየ፡ እመ፡ ይሰቲ፡ በሕልሙ።</l>
+                        <l n="2">ወኢይባልሖ፡ እምጽምዕ፡ ለሐሊበ፡ ዕጐልት፡ ጣዕሙ።</l>
+                        <l n="3">ልምላሜ፡ ጕርዔሁ፡ ያየብስ፡ ዳእሙ።</l>
+                        <l n="4">ከማሃ፡ ትድላ፡ ዓለሙ።</l>
+                        <l n="5">ዘኢረከባ፡ ይቴክዝ፡ ወዘረከባ፡ ይፃሙ።</l>
+                        <l n="6">እምዝኒ፡ መንገለ፡ ዝኒ፡ ለለጌሠሙ፡ ወአሙ።</l>
+                        <l n="7">ትትፋለስ፡ ለለጊዜ፡ ወለዐርካኒሃ፡ ተኃድጎሙ።</l>
+                        <l n="8">በእንተ፡ ዝኒ፡ ሐነጸ፡ ሀገረ፡ ነፍሱ፡ በደሙ።</l>
+                        <l n="9"><persName ref="PRS9162TaklaHa">ተክለ፡ ሃይማኖት፡</persName> አንበሳ፡ መስዕ፡ ዘ<placeName ref="INS0341DL">ደብረ፡ ሊባኖስ፡</placeName> ገዳሙ።</l>
+                        <l n="10">ወበቅድመ፡ አምላክ፡ ትቁም፡ አስተሰነዐላ፡ ለዓጽሙ።</l>
+                        <l n="11"><persName ref="PRS9162TaklaHa">ተክለ፡ ሃይማኖት፡</persName> አንበሳ፡ መስዕ፡ ዘ<placeName ref="INS0341DL">ደብረ፡ ሊባኖስ፡</placeName> ገዳሙ።</l>
+                    </ab>                    
+               
+            </div>
+            <div type="translation">
+                <div type="textpart" xml:lang="en">
```

Comment:
Is it useful to note, whether the translation is from Denis Nosnitsin or from Afawarq?

Reply:
I considered this.